certbot/account.py

```diff
@@ -74,7 +74,8 @@ def slug(self):
             self.meta.creation_dt), self.meta.creation_host, self.id[:4])
 
     def __repr__(self):
-        return "<{0}({1})>".format(self.__class__.__name__, self.id)
+        return "<{0}({1}, {2}, {3})>".format(
+            self.__class__.__name__, self.id, self.regr, self.meta)
```

Comment:
nit: As you probably already know, normally it's good practice to make the `repr` of an object be as close to the constructor call necessary to recreate that object as possible. While we don't want to do exactly that here because the `repr` would include the private key, we can keep the order the same as the constructor by swapping `id` and `regr` here.